src/index.css

```diff
@@ -1,264 +1,27 @@
-@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@200;300;400&display=swap');
-
-* {
-  padding: 0;
-  margin: 0;
-  box-sizing: border-box;
-}
-
-html {
-  font-size: 62.5%;
-}
-
 body {
```

Comment:
- [ ] As the previous reviewer requested, please remove the horizontal scrolling from your body element to match the given design and also your containers should have more spacings around as I marked below. Your containers have 100% width and no spacing: 
> 
![image](https://github.com/Oscah01/the-book-store/assets/91486791/f8d91241-831e-4371-8a06-d22275c65dca)

> your design: 
![image](https://github.com/Oscah01/the-book-store/assets/91486791/dd86f8c6-8763-4264-a8b7-ed4549ebcb8f)